src/backend/aspen/database/models/usergroup.py

```diff
@@ -114,8 +112,7 @@ class User(idbase, DictMixin):  # type: ignore
     )
     gisaid_submitter_id = Column(String, nullable=True, default=None)
 
-    group_id = Column(Integer, ForeignKey(Group.id), nullable=False)
-    group = relationship("Group", back_populates="members")  # type: ignore
+    group_id = Column(Integer, ForeignKey(Group.id), nullable=True)
```

Comment:
don't you still want the Group relationship above?

Reply:
This column and the relationship it represents are going away, so we don't need either.